tests/integration/test.replication.js

```diff
@@ -2643,15 +2759,13 @@ adapters.forEach(function (adapters) {
       var targetPut =  target.put;
       target.put = putter;
 
-      var changes = source.changes;
-      source.changes = function (opts) {
+      interceptChanges(source, function(opts) {
```

Comment:
Travis complains about this one, `function(opts)` vs. `function (opts)`

Reply:
Thanks :¬)